Source/WebCore/layout/formattingContexts/grid/TrackSizingAlgorithm.cpp

```diff
@@ -27,12 +27,25 @@
 #include "TrackSizingAlgorithm.h"
 
 #include "NotImplemented.h"
+#include "StyleGridTrackBreadth.h"
 #include "TrackSizingFunctions.h"
 #include <wtf/Vector.h>
 
 namespace WebCore {
 namespace Layout {
 
+struct FlexTrack {
+    size_t trackIndex;
+    Style::GridTrackBreadth::Flex flexFactor;
+    LayoutUnit baseSize;
+    LayoutUnit growthLimit;
+
+    FlexTrack(size_t index, Style::GridTrackBreadth::Flex factor, LayoutUnit base, LayoutUnit growth)
+        : trackIndex(index), flexFactor(factor), baseSize(base), growthLimit(growth)
```

Comment:
The usual style is to put each of these on their own line.

```suggestion
        : trackIndex(index)
        , flexFactor(factor)
        , baseSize(base)
        , growthLimit(growth)
```

Reply:
Done, thanks for the review!